hashin.py

```diff
@@ -53,7 +55,7 @@ def _download(url, binary=False):
     return r.read().decode(encoding)
 
 
-def run(spec, file, algorithm, verbose=False):
+def run(spec, file, algorithm, python_versions=[], verbose=False):
```

Comment:
Mutable default argument isn't safe. 
It needs to be `python_versions=None` followed by...
```python
python_versions = python_versions or []
```